source/_components/device_tracker.bt_smarthub.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "BT Smart Hub"
+description: "Instructions on how to integrate BT Smart Hub router into Home Assistant."
+date: 2018-10-05 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
+ha_category: Presence Detection
+---
+
+
+This platform offers presence detection by looking at connected devices to a [BT Smart Hub](https://en.wikipedia.org/wiki/BT_Smart_Hub) based router.
+This router is sometimes referred to as the BT Home Hub 6.
+
+To use a BT Smart Hub router in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: bt_smarthub
+    host: 192.168.1.254
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration